Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -245,8 +245,16 @@ private extension TabScrollingController {
     }
 
     func animateToolbarsWithOffsets(_ animated: Bool, duration: TimeInterval, headerOffset: CGFloat, footerOffset: CGFloat, alpha: CGFloat, completion: ((_ finished: Bool) -> Void)?) {
+        guard let scrollView = scrollView else { return }
+        let initialContentOffset = scrollView.contentOffset
+
+        // If this function is used to fully animate the toolbar from hidden to shown, keep the page from scrolling by adjusting contentOffset.
```

Comment:
Do you think adding a a (why?) here would be helpful? Explaining that we update these insets in order to avoid jank when loading a new url while the toolbars are hidden?

Reply:
Yeah, the more comments/explanation in this file the better.